core/src/main/resources/static/js/connectorOverview.js

```diff
@@ -526,9 +526,9 @@ app.controller("connectorOverviewCtrl", function($scope, $http, $location, $wind
             params: {'connectornamesearch' : topicSelected
              }
 		}).success(function(output) {
-		    if(output.connectorExists == true){
+		    if(output.connectorExists === true){
 		        $scope.resultBrowse = output.aclInfoList;
-            	$scope.connectorOverview = output.topicInfoList;
+            	$scope.connectorOverview = output.connectorInfoList;
             	$scope.promotionDetails = output.promotionDetails;
```

Comment:
I didnt see the change in the html to point to connectorInfoList from topicInfoList is this correcting an existing defect?

Reply:
In html, we only refer to $scope.connectorOverview, hence no changes in html

Reply:
Gotcha I read that wrong